newPost.sh

```diff
@@ -0,0 +1,46 @@
+read -p "Title: " title
+read -p "Date [$(date +%Y-%m-%d)]: " date
+read -p "Categories (space-separated): " categories
+read -p "Description: " description
+read -p "Post Image URL (full size): " img
+read -p "Post Image URL (smaller): " img_sm
+read -p "Author Name: " author_name
+read -p "Author Image URL: " author_img
+read -p "Author Bio: " author_bio
+read -p "Author E-Mail: " author_email
+read -p "Author GitHub URL: " author_github
+read -p "Author LinkedIn URL: " author_linkedin
+
+if [ -z "$date" ] # Default to current date
+then
+    date=$(date +%Y-%m-%d)
+fi
+
+title_url=$(echo "$title" | tr " " "-" | tr '[:upper:]' '[:lower:]')
+
+function add {
```

Comment:
It would be cool if you expanded this function to take multiple parameters such that you could do something like the following:

```
add "Title" "" "title"
add "Date Created" "$(date +%Y-%m-%d)" "date"
```
Where the parameters are in the following order:
1. The field title you are asking for input for
2. The default value of the field "" for empty
3. The field yaml key

Then optionally you could add a fourth True/False parameter that is whether the field is required or not. Because a title and date created would be required for the post, but a author-linked in may not be.